charts/ssi-credential-issuer/templates/cronjob-data-deletion-app.yaml

```
@@ -72,8 +72,8 @@ spec:
             {{- end }}
             - name: "EXPIRY__EXPIREDVCSTODELETEINMONTH"
               value: "{{ .Values.credentialExpiry.expiry.expiredVcsToDeleteInMonth }}"
-            - name: "EXPIRY__INACTIVEVCSTODELETEINWEEKS"
-              value: "{{ .Values.credentialExpiry.expiry.inactiveVcsToDeleteInWeeks }}"
+            - name: "EXPIRY__INACTIVEVCSTODELETEINDAYS"
+              value: "{{ .Values.credentialExpiry.expiry.inactiveVcsToDeleteInDAYS }}"
```

Please change to `Values.credentialExpiry.expiry.inactiveVcsToDeleteInDays`

```suggestion
              value: "{{ .Values.credentialExpiry.expiry.inactiveVcsToDeleteInDays }}"
```